README.md

```diff
@@ -6,3 +6,7 @@ worlds-simplest-csharp-wpf-mvvm-example
 The World's Simplest C# WPF MVVM Example as described [here](http://www.markwithall.com/programming/2013/03/01/worlds-simplest-csharp-wpf-mvvm-example.html).
 
 A C# 6 version of the code can be found in the [C#6.0 branch](https://github.com/MarkWithall/worlds-simplest-csharp-wpf-mvvm-example/tree/C%236.0).
+
+---
+
+I didn't modify the structure of this program from the original creator, I just added a whole bunch of comments to better describe it. As a newbie to C#, this turned out to be a great example to learn from, but it was way harder than it should've been to get a handle on how it worked since there were no comments in the actual code. This is my attempt to remedy that. PR's welcome!
```

Comment:
This comment in the README isn't applicable to this repository and should be removed from the PR.